### ISSUE TITLE
conditional_binding_cascade false positive 

### ISSUE THREAD

**Issue by ReedD** [realm/SwiftLint#645]
@norio-nomura I believe this is from your most recent commit. The following triggers a `conditional_binding_cascade` warning for repeated `let` statements. However `c` is a obviously a `var` not a `let`.

```
if let a = b, 
   var c = d {
    // Some code...
}
```

**Comment by norio-nomura** [comment]
Duplicate of #642
